test/org/rascalmpl/util/maven/proxyhomedir/.m2/settings.xml

```diff
@@ -0,0 +1,15 @@
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 https://maven.apache.org/xsd/settings-1.0.0.xsd">
+
+    <proxies>
+        <proxy>
+            <id>testproxy</id>
+            <active>true</active>
+            <protocol>http</protocol>
+            <host>91.107.227.58</host>
```

Comment:
can this not be localhost? instead of your IP address? (might want to scrub this using git rebase ;) )